msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1043,6 +1043,16 @@ Required if the handler type is `openPage`. Object containing metadata of the pa
 |`PageId`| String | |  | Maps to the `EntityId` of the static tab. |
 |`SubPageId`| String | |  | Maps to the `SubEntityId` of the static tab. |
 
+## intuneInfo
+
+**Optional** &ndash; Object
+
+Properties related to app support for Microsoft Intune.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`supportedMobileAppManagementVersion`| String | 64 characters |  | Supported Microsoft Intune mobile app management (MAM) version that the app is compliant with. Expected value is a single version number, in the form of `integer.integer` (for example, `1.2`), that represents the maximum version of support to which this app attests. If no value is specified, the app does not attest to be mobile app management compliant. |
```

Comment:
```suggestion
|`supportedMobileAppManagementVersion`| String | 64 characters |  | Supported Microsoft Intune Mobile App Management (MAM) version. The value is a single version number in the `integer.integer` format, such as `1.2`, indicating the highest level of support the app confirms. If no value is provided, the app doesn't comply with mobile app management. |
```

Reply:
Hi @erikadoyle,

What does "app is compliant with" here mean? If we remove it, does it change the meaning of the description?

If we are trying to say that "an app is compliant if it has a MAM version", I believe we don't need to mention it in the description in the manifest. Please correct me if my understanding is incorrect.

thanks,
Yash